netci.groovy

```diff
@@ -231,6 +231,27 @@ commitPullList.each { isPr ->
   addRoslynJob(myJob, jobName, branchName, isPr, triggerPhraseExtra, triggerPhraseOnly)
 }
 
+// VS Integration Tests
+commitPullList.each { isPr ->
+  ['debug', 'release'].each { configuration ->
+    ['vs-integration'].each { buildTarget ->
+      def jobName = Utilities.getFullJobName(projectName, "windows_${configuration}_${buildTarget}", isPr)
+      def myJob = job(jobName) {
+        description("Windows ${configuration} tests on ${buildTarget}")
+        steps {
+          batchFile(""".\\build\\scripts\\cibuild.cmd ${(configuration == 'debug') ? '-debug' : '-release'} -testVsi""")
```

Comment:
Just do -$configuration

Reply:
The current pattern matches elsewhere in the file. I'm trusting that overall this is just a revert of a previous change that decided to use deletion instead of commenting.